util/reformat-yaml/reformat.go

```diff
@@ -0,0 +1,120 @@
+package main
```

Comment:
I can't comment on that actual file, but you shouldn't check in the actual `reformat` binary — it won't work on every machine (eg on different OSes). Anyone who wants to run it should build it themselves.